### PR TITLE
remove unnecessary this binding for helper (ONLY needed for callbacks)

### PR DESCRIPTION
**Pull request by mamute**

### CODE REVIEW

File: client/app/components/RouterCommentScreen.jsx [hunk 1]
```diff
@@ -10,6 +11,12 @@ function select(state) {
 }
 
 class RouterCommentScreen extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    _.bindAll(this, '_renderNotification');
```
Comment (justin808) [inline client/app/components/RouterCommentScreen.jsx]: @mamute Can you check this is actually required, meaning a binding of this.

The reason I ask is that it looks like this._renderNotifcation is simply a helper that returns either null or a react component.

I suspect that the fix is that that the `::` was not needed.

**Comment by mamute** [comment]
@justin808 you're right, i've checked and fixed

**Comment by justin808** [comment]
Thanks. @alexfedoseev FYI...

@mamute I updated the title.

**Comment by mamute** [comment]
:+1:

**Pull request merged**